src/components/Main.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+
+class Main extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        let list = this.props.items.map((el, index) => {
+            return (
+                <div className="view">
+                    <li className="item" key={index}>
```

Comment:
it's not a good practice to pass `index` as `key`
You should have field `id` for each item